### ISSUE TITLE
Why are VAOs not used?

### ISSUE THREAD

**Issue by vangelov** [jagenjo/litegl.js#14]
I might be missing something, but it seems the library doesn't use VAOs. Why not?

**Comment by jagenjo** [comment]
yeah, I never implemented it. I should give it a try, AFAIK improves the performance but dont know how much.

**Comment by vangelov** [comment]
I don't have any data about that unfortunately, but it seems to be the standard way when using WebGL2. For WebGL1 you'll have to use an extension (if available) however.
